float64[] poseLeft
float64[] poseRight
float64 jawLeft
float64 jawRight
int32 clutchLeft
int32 clutchRight 